contracts/integrations/test/actors/pool_operator.ts

```diff
@@ -63,13 +77,51 @@ export function PoolOperatorMixin<TBase extends Constructor>(Base: TBase): TBase
             newOperatorShare: number,
         ): Promise<TransactionReceiptWithDecodedLogs> {
             const stakingContract = this.actor.deployment.staking.stakingWrapper;
-            this.operatorShares[poolId] = newOperatorShare;
             return stakingContract.decreaseStakingPoolOperatorShare.awaitTransactionSuccessAsync(
                 poolId,
                 newOperatorShare,
                 { from: this.actor.address },
             );
         }
+
+        private _getOperatorPoolIds(simulationEnvironment: SimulationEnvironment): string[] {
+            const operatorPools = _.pickBy(
+                simulationEnvironment.stakingPools,
+                pool => pool.operator === this.actor.address,
+            );
+            return Object.keys(operatorPools);
+        }
+
+        private async *_validCreateStakingPool(
```

Comment:
Is there a plan to include `invalid` generators?

Reply:
yes, definitely. Will be in the next PR